catroid/src/main/java/org/catrobat/catroid/CatroidApplication.java

```diff
@@ -47,6 +48,7 @@
 	public void onCreate() {
 		super.onCreate();
 		CrashReporter.initialize(this);
+		CloudMessaging.initialize(this);
```

Comment:
Hmm. With current configuration (checkbox is false by default) I guess not. But IMO the checkbox value should be set to true by default (because generally, users don't go to the settings tab unless required.) 

So, if you want me to remove it then let me know.